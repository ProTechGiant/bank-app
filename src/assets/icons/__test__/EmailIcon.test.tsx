import { create } from "react-test-renderer";

import { EmailIcon } from "../EmailIcon";

describe("icon/EmailIcon", () => {
  it("renders a EmailIcon", () => {
    const result = create(<EmailIcon />);

    expect(result).toMatchSnapshot();
  });
});
