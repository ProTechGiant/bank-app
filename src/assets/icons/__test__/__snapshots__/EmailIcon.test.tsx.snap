// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon/EmailIcon renders a EmailIcon 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="44"
  bbWidth="46"
  fill="none"
  focusable={false}
  height={44}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 44,
        "width": 46,
      },
    ]
  }
  vbHeight={44}
  vbWidth={46}
  width={46}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M31.334 15.334H14.667v13.333h16.667V15.333zm-1.667 3.333L23 22.833l-6.666-4.166V17L23 21.167 29.667 17v1.667z"
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
